WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Split Shipments/WooShippingSplitShipmentsDetailView.swift

```diff
@@ -8,23 +8,28 @@ struct WooShippingSplitShipmentsDetailView: View {
 
     var body: some View {
         NavigationView {
-            ScrollView {
-                VStack(alignment: .leading, spacing: Layout.contentPadding) {
-                    AdaptiveStack(horizontalAlignment: .leading) {
-                        Text(viewModel.itemsCountLabel)
-                            .headlineStyle()
-                        Spacer()
-                        Text(viewModel.itemsDetailLabel)
-                            .foregroundStyle(Color(.textSubtle))
-                    }
+            ZStack(alignment: .bottom) {
```

Comment:
I think we should use a VStack instead, because otherwise the notice would cover the scroll view when it's long enough:

<img src="https://github.com/user-attachments/assets/929fd8e9-5ec6-49e3-97b9-bb939d5daaf5" width=320 />